src/redux/selectors/exposedSelectors.js

```diff
@@ -99,7 +99,9 @@ export const isEmbedPrintSupported = state => {
     window.navigator.userAgent.indexOf('Chrome') > -1 &&
     window.navigator.userAgent.indexOf('Edge') === -1;
   const isPDF = getDocumentType(state) === workerTypes.PDF;
-  return isPDF && isChrome && state.viewer.useEmbeddedPrint;
+  // return isPDF && isChrome && state.viewer.useEmbeddedPrint;
+  // TODO remove this later
+  return false;
```

Comment:
Need to revert this change.

Reply:
i will revert it as the very last thing before i merge.
I want to be able to test out any new code changes suggested in this PR to make sure it doesn't break.

Reply:
@mparizeau-pdftron when you think everything is good. i will fix this line.

Reply:
@mparizeau-pdftron i have reverted this line back to what it used to be.